apps/io-wallet-user-func/src/infra/pid-issuer/client.ts

```diff
@@ -0,0 +1,84 @@
+import { PidIssuerApiClientConfig } from "@/app/config";
+import { CredentialRepository } from "@/credential";
+import { FiscalCode } from "@pagopa/ts-commons/lib/strings";
+import * as TE from "fp-ts/lib/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+import { Agent, RequestInit, fetch } from "undici";
```

Comment:
I see this new dependency has been added. Maybe we'll try to use the `fetch` of `undici` for the other flows too, what do you think? Obviously in a separate PR

Reply:
actually I wanted to see if it's possible to use node's native `fetch` here as well instead of importing `undici` for mTLS 😄